tests/test_bgp_neighbor_af.rb

```diff
@@ -150,15 +233,17 @@ def test_props_bool
   def props_bool(af, dbg)
     # These properties have simple boolean states. As such we can use a common
     # set of tests to validate each property.
+    # Common to both XR and nexus
     props = [
       :as_override,
-      :disable_peer_as_check,
       :next_hop_self,
-      :next_hop_third_party,
-      :suppress_inactive,
     ]
 
-    props = [:disable_peer_as_check] if dbg.include?('l2vpn/evpn')
+    if platform == :nexus
+      props << :next_hop_third_party
+      props << :suppress_inactive
+      props = [:disable_peer_as_check] if dbg.include?('l2vpn/evpn')
```

Comment:
OK. Consider adding a comment and/or negative tests as this is not obvious just looking at the code.